src/main/java/org/testng/internal/PackageUtils.java

```diff
@@ -30,7 +30,7 @@
   private static String[] s_testClassPaths;
 
   /** The additional class loaders to find classes in. */
-  private static final List<ClassLoader> m_classLoaders = new Vector<ClassLoader>();
+  private static final List<ClassLoader> m_classLoaders = new Vector<>();
```

Comment:
yep. it's not true already.
"<source>1.7</source>"
https://github.com/cbeust/testng/blob/master/pom.xml#L161